pilot/pkg/features/pilot.go

```diff
@@ -322,4 +322,9 @@ var (
 			"It is safe to disable it if you are quite sure you don't need this feature").Get()
 	InjectionWebhookConfigName = env.RegisterStringVar("INJECTION_WEBHOOK_CONFIG_NAME", "istio-sidecar-injector",
 		"Name of the mutatingwebhookconfiguration to patch, if istioctl is not used.")
+
+	SpiffeBundlePaths = env.RegisterStringVar("SPIFFE_BUNDLE_PATHS", "",
+		"The paths to the SPIFFE bundle endpoints, in JSON format. For example: "+
+			"[{\"trustdomain\": \"foo\", \"url\": \"https://exampleurl1.com:123/path\"},"+
```

Comment:
Ok for the env ( experimental, etc ) - but when we add it to ProxyConfig we should use a simpler format, like:
spiffe_roots:
  trustdomain1: URL1
  trustdomain2: URL2

Reply:
Yes, I was thinking about this. But it could be confusing if we don't define it well. For example ":" can't be used as the delimiter since it's allowed in URLs.